### PR TITLE
add a11y labeling to Trip Viewer

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
- Adds trip description
- Adds aria labels for stops related to trip
- Re-orders trip viewer list items for screen readers

### CODE REVIEW

File: lib/components/viewers/trip-viewer.js
Comment (miles-grant-ibigroup) [inline lib/components/viewers/trip-viewer.js:190]: I don't think we need the title as well!

File: lib/components/viewers/trip-viewer.js
Comment (amy-corson-ibigroup) [reply lib/components/viewers/trip-viewer.js:51]: I'm not proud of this one

File: lib/components/viewers/trip-viewer.js
Comment (amy-corson-ibigroup) [reply lib/components/viewers/trip-viewer.js]: Hmm I tried a different approach, without the border, tightening the text block and adding a subtle background color. I'm not sure it's quite there, but maybe it's in the right direction? Let me know!

Comment (miles-grant-ibigroup) [reply lib/components/viewers/trip-viewer.js]: This is it! I love it. My only 2 comments on it at this point are to match the color with the highlights on the blobs (I'd adjust the blob highlight color -- I like yours more)

and maybe adding
```css
border-radius: 5px;
box-shadow: 2px 2px 5px 1px rgba(0, 0, 0, 0.02);
```
to the box? What do you think

File: lib/components/viewers/trip-viewer.js
Comment (miles-grant-ibigroup) [inline lib/components/viewers/trip-viewer.js:47]: We can probably fully remove this!

File: lib/components/viewers/trip-viewer.js
Comment (miles-grant-ibigroup) [inline lib/components/viewers/trip-viewer.js:199]: I think I'd almost prefer an inline style here rather than this extra prop, but will leave it to the second reviewer to decide!

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks so good! Thanks for your work on this. Curious for your thoughts on the proposed css, otherwise approved pending 2 tiny nitpicks!

File: lib/components/viewers/trip-viewer.js
Comment (amy-corson-ibigroup) [reply lib/components/viewers/trip-viewer.js:47]: whoops

File: lib/components/viewers/trip-viewer.js
Comment (amy-corson-ibigroup) [reply lib/components/viewers/trip-viewer.js]: LOVE this! I added these, and a TINY bit of padding, and I think it's looking pretty good!

Changing the highlight color on the blobs to match the description is tricky because it's unfortunately barely visible. The description background is decoration, but the highlight is at least in part trying to communicate information? I split the difference and made the opacity `0.5` but it may just look inconsistent, so I'm very open to feedback!

File: lib/components/viewers/trip-viewer.js
Comment (binh-dam-ibigroup) [inline lib/components/viewers/trip-viewer.js:128]: Good idea!

You might be able to surround the contents with `<strong>` directly instead of passing a `<strong>` tag. 

Add a TODO to add links to the entry in the list of stops, so that when people navigate, they can see/hear the time they will reach the boarding and arriving stops, and also access the stop viewer links without having to go through all the stops not on the trip.

File: lib/components/viewers/trip-viewer.js
Comment (binh-dam-ibigroup) [inline lib/components/viewers/trip-viewer.js:199]: `aria-label` does not seem to be picked up on a regular `<li>`, probably because it is not interactive. I think it is better to insert a `<span>` before or after the stop name that is not visible to sighted users (e.g. use zero-width/height and/or clip).

File: lib/components/viewers/trip-viewer.js
Comment (binh-dam-ibigroup) [inline lib/components/viewers/trip-viewer.js:43]: Or use `<Alert bsStyle="info">` (as long as we are using bootstrap)

File: lib/components/viewers/viewers.css
Comment (binh-dam-ibigroup) [inline lib/components/viewers/viewers.css:458]: While you are at it... sort CSS props.

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
The labels for "Trip begins/ends here" are not showing in ARIA dev tools for some reason, please fix that (see comment).

File: lib/components/viewers/trip-viewer.js
Comment (amy-corson-ibigroup) [reply lib/components/viewers/trip-viewer.js:43]: Agree it looks very similar to Bootstrap, are we trying to move away from Bootstrap long term?

Comment (miles-grant-ibigroup) [reply lib/components/viewers/trip-viewer.js:43]: Agreed I think we should avoid adding any more bootstrap

Comment (miles-grant-ibigroup) [reply lib/components/viewers/trip-viewer.js:43]: But perhaps making this `TripDescription` thing into a custom `Alert` component and putting in utils might be a good middle ground!

File: lib/components/viewers/trip-viewer.js
Comment (amy-corson-ibigroup) [reply lib/components/viewers/trip-viewer.js:199]: Smart, thank you!

File: lib/components/viewers/trip-viewer.js
Comment (amy-corson-ibigroup) [reply lib/components/viewers/trip-viewer.js]: added a component to utils in fb0e0f4b1006719b8ae4fbf1f1e3e3f80d95c2e7 !

File: i18n/en-US.yml
Comment (binh-dam-ibigroup) [inline i18n/en-US.yml:557]: ❤️

File: lib/components/util/alert.tsx
Comment (binh-dam-ibigroup) [inline lib/components/util/alert.tsx:19]:
```suggestion
const Alert = styled.div`
  background-color: rgba(19, 193, 193, 0.2);
  border-radius: 5px;
  box-shadow: 2px 2px 5px 1px rgba(0, 0, 0, 0.02);
  font-size: 12px;
  margin-bottom: 1em;
  padding: 0.75em 1em;
`

export default Alert
```

File: lib/components/viewers/trip-viewer.js
Comment (binh-dam-ibigroup) [inline lib/components/viewers/trip-viewer.js:50]: For spans, you need `display: block` or `display: inline-block` for the text to be clipped.

File: lib/components/viewers/trip-viewer.js
Comment (binh-dam-ibigroup) [inline lib/components/viewers/trip-viewer.js:187]: You want to keep the id value together with `<FormattedMessage>` (or with intl.formatMessage), so that the formatjs CLI tool can pick up the string to translate. (In other words, don't use dynamic ids - the exception is for using ids defined in config.)

File: lib/components/viewers/trip-viewer.js
Comment (binh-dam-ibigroup) [reply lib/components/viewers/trip-viewer.js:202]: I favor inlining the `80%` right into the CSS.

File: lib/components/viewers/trip-viewer.js
Comment (binh-dam-ibigroup) [inline lib/components/viewers/trip-viewer.js:207]: Keep `<FormattedMessage>` and the value for `id` together.

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Some more cleanup needed, please.

File: lib/components/viewers/trip-viewer.js
Comment (amy-corson-ibigroup) [reply lib/components/viewers/trip-viewer.js:50]: The overflow property is wrong too, what a mess, thank you!!

File: lib/components/viewers/trip-viewer.js
Comment (amy-corson-ibigroup) [reply lib/components/viewers/trip-viewer.js]: Good to know! Thanks! Fixed in 67caf50f05cfa00f575984c8559b788eb79f36e8

File: lib/components/util/alert.tsx
Comment (binh-dam-ibigroup) [inline lib/components/util/alert.tsx:11]: Rename to `alert.ts` now that this file no longer has markup.

**Review by binh-dam-ibigroup — APPROVED** [review]
Thanks for the changes!

**Pull request merged**
